### PR TITLE
Add license to packages

### LINKED ISSUE #16: Packaging license file
Would be nice to add `LICENSE.txt` to `MANIFEST.in` so that the license file ships with `sdist`s and other packages. Bonus points if it is added to [`setup.cfg` for wheels]( https://wheel.readthedocs.io/en/stable/user_guide.html#including-license-files-in-the-generated-wheel-file ). This will also make it easier for downstream packages to ship the license file (e.g. a conda-forge package). ( https://github.com/jacobtomlinson/jupyterlab-nvdashboard/issues/11 )

### PR DESCRIPTION
**Pull request by jacobtomlinson**
Add license file to package and wheels.

Closes #16 

cc @jakirkham

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[metadata]
+license_files =
+   LICENSE.txt
```
Comment (jakirkham) [inline setup.cfg]: Could put these on the same line I think (or maybe GitHub is rendering this weird for me)?

**Comment by jakirkham** [comment]
Looks good. One comment above. Thanks @jacobtomlinson !

File: setup.cfg (hunk 1 above)
Comment (jacobtomlinson) [reply setup.cfg]: I think this is a list of one item. There was a second item from the docs you linked to that I removed.

Comment (jakirkham) [reply setup.cfg]: It works fine if you have only one license and include it on the same line. Here's [an example]( https://github.com/dask/dask/blob/fbc373741264244102efc25d53893d0d83e669a6/setup.cfg#L48-L49 ).

**Pull request merged**